webknossos/webknossos/dataset/dataset.py

```diff
@@ -117,6 +118,61 @@
 # This regex matches any character that is not allowed in a layer name.
 _UNALLOWED_LAYER_NAME_CHARS = re.compile(r"[^A-Za-z0-9_$@\-\.]")
 
+_VOXEL_UNITS = {
```

Comment:
maybe move to a different file? I feel like dataset.py is already a bit too big :sweat_smile:

Reply:
I made a unit.py file within the dataset folder, I am not sure if it is a really fitting place, but it is similar to data_format.py

Reply:
When users want to create a new dataset and supply a voxel size with length unit (not via cli), they will need to import those classes, right? How are they exposed? Do they need to be forwarded in any init files?